Source/AST/Styling/Stylers/DownStyler.swift

```diff
@@ -61,12 +61,17 @@ open class DownStyler: Styler {
     open func style(blockQuote str: NSMutableAttributedString, nestDepth: Int) {
         let stripeAttribute = QuoteStripeAttribute(level: nestDepth + 1, color: colors.quoteStripe, options: quoteStripeOptions)
 
+        str.addAttributes([
+            .font: fonts.quote,
```

Comment:
I believe this will override existing fonts in the quote, which means that quotes would no longer preserve inline styles such as strong, emphasis and code. Is this actually the case?

Reply:
You could be correct, I might need to apply a similar piece of logic to this aswell

Reply:
Updated to use a similar function as headings.